src/components/filter/EventsFilterPanel.tsx

```diff
@@ -45,6 +45,14 @@ function getDefaultCurrentFilterValues(filter: EventsFilter | null) {
 		: null;
 }
 
+const priority = [
+	'attachedEventIds-negative',
```

Comment:
attachedEventIds?  did you mean messages.
remove all this values and replace them with
keys of EventsFilter, same with Messages but there you will use keys of MessageFilterState

and sorting suppose to happen at line 125
`const filterNames = getObjectKeys(filter).sort(sortCb);`